.circleci/config.yml

```diff
@@ -410,6 +410,29 @@ commands:
       # Allow this to fail, but it is useful for debugging.
       - run: sh -c "pip check || true"
 
+  setup-docker-env:
+    description: Set up and create docker env.
+    parameters:
+      image:
+        type: string
+      image-name:
+        type: string
+        default: "test_container"
+      bind-mount:
+        type: string
+        default: "$PWD"
+      extras-requires:
+        type: string
+        default: ""
+    steps:
+      - run: docker run -v <<parameters.bind-mount>>:/usr/src/project --gpus=all -it -d --name <<parameters.image-name>> --workdir /usr/src/project <<parameters.image>>
```

Comment:
_This is kind of a nit, but it'll get weird if we choose this pattern now and stick to it everywhere, so..._

Let's stick to the [Filesystem Hierarchy Standard](https://refspecs.linuxfoundation.org/FHS_3.0/fhs.html) when picking file locations.  The `/usr` directory is meant to be [read-only](ttps://refspecs.linuxfoundation.org/FHS_3.0/fhs.html#purpose18) in most cases, so it's a weird place ot mount this stuff.

Instead, let's put everything in `/mnt`, which is meant for [temporarily mounted files](https://refspecs.linuxfoundation.org/FHS_3.0/fhs.html#mntMountPointForATemporarilyMount), which is closer to what we're doing here.